WcaOnRails/app/models/competition.rb

```diff
@@ -48,6 +51,7 @@ class Competition < ActiveRecord::Base
     cellName
     showAtAll
     isConfirmed
+    eventSpecs
```

Comment:
I've left the column there in this PR so it's easy to rollback. I was planning on dropping the column in a separate PR after this is implemented.